client/my-sites/upgrades/trials/data/free-trial-cart.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import cartValues, { cartItems } from 'lib/cart-values';
+import observe from 'lib/mixins/data-observe';
+import TransactionStore from 'lib/transaction/store';
+import StoreConnection from 'components/data/store-connection';
+import { resetTransaction, setPayment } from 'lib/upgrades/actions';
+import storeTransactions from 'lib/store-transactions';
+
+const FreeTrialCartData = React.createClass( {
+	mixins: [ observe( 'sites', 'products' ) ],
+
+	getInitialState: function() {
+		return { cart: {} };
```

Comment:
Is calling this "cart" a good idea, since this isn't a cart in the same way that our other "cart"s are (unless I am misunderstanding)